content/en/docs/components/pipelines/sdk/connect-api.md

```diff
@@ -53,7 +53,93 @@ because it requires authentication. Refer to distribution specific documentation
+Read more about **configurations** on [Jupyter notebook server](/docs/components/notebooks/setup/#create-a-jupyter-notebook-server-and-add-a-notebook).
+
+Note, Kubeflow `kfp.Client` expects token either in `KF_PIPELINES_SA_TOKEN_PATH` environment variable or mounted to `/var/run/secrets/kubeflow/pipelines/token`. Do not change these values in the manifest. Similarly, `audience` should not be modified as well. No additional setup is required to refresh tokens.
+
+Setup is similar for other cases like e.g. **cron tasks or in-cluster CI/CD**: Configure your Pod with token in either `KF_PIPELINES_SA_TOKEN_PATH` or `/var/run/secrets/kubeflow/pipelines/token` and `kfp.Client` will gain access to Kubeflow Pipelines.
+
+##### Managing cross-namespaces access to Kubeflow Pipelines API
+
+The setup can be extended to allow other namespaces to access Kubeflow Pipelines API through your namespace's Juputer notebook server.
+
+Note, examples below assume that `namespace-1` is a namespace (profile) that will  be granted access to Kubeflow Pipelines API 
+through `namespace-2` Jupyter notebook server. `namespace-2` is already configured to access Kubeflow Pipelines API.
+
+The access can be achieved in two ways:
+
+* **With additional RBAC settings.**
+
+  This option requires that only `namespace-2` has to have `PodDefault` manifest configured.
+
+  Access is granted by giving `namespace-1:ServiceAccount/default-editor` the `ClusterRole/kubeflow-edit` in `namespace-2`:
+
+  ```
+  apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: kubeflow-edit-namespace-1
+    namespace: namespace-2
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: kubeflow-edit
+  subjects:
+  - kind: ServiceAccount
+    name: default-editor
+    namespace: namespace-1
+  ```
+
+* **By sharing access to the other profile.**
+
+  In this scenario, both `namespace-1` and `namespace-2` require `PodDefault` to be properly configured. Access is granted by `namespace-2` adding `namespace-1` as a  
+  [contributor](https://www.kubeflow.org/docs/components/multi-tenancy/getting-started/#managing-contributors-through-the-kubeflow-ui).
+
+##### How in-cluster authentication works
+
+Authentication uses ServiceAccountToken 
+[projection](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) where token with limited lifetime, that can be verified by Kubeflow Pipelines API, is being injected into (e.g. Jupyter notebook's) Pod.
+
+More details about PodDefault can be found [here](https://github.com/kubeflow/kubeflow/blob/master/components/admission-webhook/README.md).
+
+It is important to understand that `serviceAccountToken` method respects the Kubeflow Pipelines RBAC, and does not allow access beyond what the ServiceAcount running the notebook Pod has.
```

Comment:
Do you think it's helpful emphasizing on the two steps: authenticate and then authorize?

1. service account token projection allows you to authenticate as the service account (default-editor in your namespace)
2. KFP API will verify whether your authenticated identity has the right permissions to do certain actions (RBAC)

Reply:
My understanding is that projected token is separate from the Pod's (e.g. default-editor) service account. 
I've modified this paragraph to:
"Authentication uses ServiceAccountToken projection. Simply put, the token is first being injected into a Pod (e.g. Jupyter notebook's). Then Kubeflow Pipelines SDK uses this token to authorize against Kubeflow Pipelines API."

Please @Bobgy @elikatsis @andreyvelich @thesuperzapper correct me if I'm wrong.